### DISCUSSION TITLE
Ist es möglich, den ESP32 mit CMT über Raspberry Pi 4 mit Strom zu versorgen

### DISCUSSION THREAD

**Discussion by Jamie-Jameson** [tbnobody/OpenDTU#1057] [Q&A]
Hallo,

derzeit betreibe ich einen Rasberry Pi 4 sowie eine OpenDTU mit CMT Chip über 2 Netzteile. Aber eigentlich fänd ich es viel schicker, wenn ich die OpenDTU per USB / 5V / 3.3V direkt aus dem RPI mit Strom zu versorgen.

Ist das möglich? Macht das Sinn, insb bezüglich Stromverbrauch? Leider bin ich mit Elektronik eher unerfahren und hab wenig Ahnung, was geht und was nicht.

**Comment by DPO99** [comment]
Moin, 
also ich würde das nicht empfehlen. Der Raspberry ist nicht dafür gemacht groß Verbraucher zu versorgen. Strom sparen kannst damit auch nicht. Der Rpasberry 4 kann ja auch gerne mal zicken, wenn er nicht ordentlich mit Strom versorgt wird (wobei das hier vermutlich unwarscheinlich ist), ebenso die DTU, gerade wenn man auch die große Antenne benutzt, ist die Stromversorgung schon wichtig. Mein Tipp. Lass wie es ist.
